src/components/ProfileInfo/ProfileInfo.scss

```diff
@@ -0,0 +1,9 @@
+.ProfileInfo{}
+
+.Avatar {
```

Comment:
use all small kebab cases in class names.

Reply:
Okay, noted. 
Actually **.ProfileInfo** was auto generated, so I thought, that is what the convention is for classes.